diff -urN bs-0.99b2/configure.in bs-0.99b2.new/configure.in
--- bs-0.99b2/configure.in	2004-02-29 17:11:44.000000000 +0100
+++ bs-0.99b2.new/configure.in	2005-03-07 01:36:22.000000000 +0100
@@ -107,7 +107,7 @@
     UXMON_DIR=${root}/usr/share/bigsister/uxmon
     BIN_DIR=${root}/usr/share/bigsister/bin
     VAR_DIR=${root}/var/lib/bigsister
-    ETC_DIR=${root}/usr/share/bigsister/etc
+    ETC_DIR=${root}/etc/bigsister/adm
     ADM_DIR=${root}/etc/bigsister
     WEB_DIR=${root}/var/lib/bigsister/www
     CGI_DIR=${root}/usr/share/bigsister/cgi
@@ -118,7 +118,7 @@
     EX_UXMON_DIR=${ex_prefix}/usr/share/bigsister/uxmon
     EX_BIN_DIR=${ex_prefix}/usr/share/bigsister/bin
     EX_VAR_DIR=${ex_prefix}/var/lib/bigsister
-    EX_ETC_DIR=${ex_prefix}/usr/share/bigsister/etc
+    EX_ETC_DIR=${ex_prefix}/etc/bigsister/adm
     EX_ADM_DIR=${ex_prefix}/etc/bigsister
     EX_WEB_DIR=${ex_prefix}/var/lib/bigsister/www
     EX_CGI_DIR=${ex_prefix}/usr/share/bigsister/cgi
