diff -urN bs-0.99b2/configure bs-0.99b2.new/configure
--- bs-0.99b2/configure	2004-02-29 17:11:44.000000000 +0100
+++ bs-0.99b2.new/configure	2005-03-20 04:42:19.409438288 +0100
@@ -1579,7 +1579,7 @@
     UXMON_DIR=${root}/usr/share/bigsister/uxmon
     BIN_DIR=${root}/usr/share/bigsister/bin
     VAR_DIR=${root}/var/lib/bigsister
-    ETC_DIR=${root}/usr/share/bigsister/etc
+    ETC_DIR=${root}/etc/bigsister/etc
     ADM_DIR=${root}/etc/bigsister
     WEB_DIR=${root}/var/lib/bigsister/www
     CGI_DIR=${root}/usr/share/bigsister/cgi
@@ -1590,7 +1590,7 @@
     EX_UXMON_DIR=${ex_prefix}/usr/share/bigsister/uxmon
     EX_BIN_DIR=${ex_prefix}/usr/share/bigsister/bin
     EX_VAR_DIR=${ex_prefix}/var/lib/bigsister
-    EX_ETC_DIR=${ex_prefix}/usr/share/bigsister/etc
+    EX_ETC_DIR=${ex_prefix}/etc/bigsister/etc
     EX_ADM_DIR=${ex_prefix}/etc/bigsister
     EX_WEB_DIR=${ex_prefix}/var/lib/bigsister/www
     EX_CGI_DIR=${ex_prefix}/usr/share/bigsister/cgi
@@ -1622,25 +1622,25 @@
     BIN_DIR=${root}/bin
     VAR_DIR=${root}/var
     ETC_DIR=${root}/etc
-    ADM_DIR=${root}/adm
+    ADM_DIR=${root}/etc
     WEB_DIR=${root}/www
     CGI_DIR=${root}/cgi
     DOC_DIR=${root}/doc
     MAN_DIR=${root}/man
     USER_SBIN_DIR=${BIN_DIR}
     USER_BIN_DIR=${BIN_DIR}
-    RUN_DIR=${root}/adm
+    RUN_DIR=${root}/etc
     EX_UXMON_DIR=${ex_prefix}/uxmon
     EX_BIN_DIR=${ex_prefix}/bin
     EX_VAR_DIR=${ex_prefix}/var
     EX_ETC_DIR=${ex_prefix}/etc
-    EX_ADM_DIR=${ex_prefix}/adm
+    EX_ADM_DIR=${ex_prefix}/etc
     EX_WEB_DIR=${ex_prefix}/www
     EX_CGI_DIR=${ex_prefix}/cgi
     EX_DOC_DIR=${ex_prefix}/doc
     EX_USER_SBIN_DIR=${EX_BIN_DIR}
     EX_USER_BIN_DIR=${EX_BIN_DIR}
-    EX_RUN_DIR=${ex_prefix}/adm
+    EX_RUN_DIR=${ex_prefix}/etc
     EX_MAN_DIR=${ex_prefix}/man
 fi
 
diff -urN bs-0.99b2/configure.in bs-0.99b2.new/configure.in
--- bs-0.99b2/configure.in	2004-02-29 17:11:44.000000000 +0100
+++ bs-0.99b2.new/configure.in	2005-03-20 04:44:15.934723752 +0100
@@ -107,7 +107,7 @@
     UXMON_DIR=${root}/usr/share/bigsister/uxmon
     BIN_DIR=${root}/usr/share/bigsister/bin
     VAR_DIR=${root}/var/lib/bigsister
-    ETC_DIR=${root}/usr/share/bigsister/etc
+    ETC_DIR=${root}/etc/bigsister/etc
     ADM_DIR=${root}/etc/bigsister
     WEB_DIR=${root}/var/lib/bigsister/www
     CGI_DIR=${root}/usr/share/bigsister/cgi
@@ -118,7 +118,7 @@
     EX_UXMON_DIR=${ex_prefix}/usr/share/bigsister/uxmon
     EX_BIN_DIR=${ex_prefix}/usr/share/bigsister/bin
     EX_VAR_DIR=${ex_prefix}/var/lib/bigsister
-    EX_ETC_DIR=${ex_prefix}/usr/share/bigsister/etc
+    EX_ETC_DIR=${ex_prefix}/etc/bigsister/etc
     EX_ADM_DIR=${ex_prefix}/etc/bigsister
     EX_WEB_DIR=${ex_prefix}/var/lib/bigsister/www
     EX_CGI_DIR=${ex_prefix}/usr/share/bigsister/cgi
@@ -150,25 +150,25 @@
     BIN_DIR=${root}/bin
     VAR_DIR=${root}/var
     ETC_DIR=${root}/etc
-    ADM_DIR=${root}/adm
+    ADM_DIR=${root}/etc
     WEB_DIR=${root}/www
     CGI_DIR=${root}/cgi
     DOC_DIR=${root}/doc
     MAN_DIR=${root}/man
     USER_SBIN_DIR=${BIN_DIR}
     USER_BIN_DIR=${BIN_DIR}
-    RUN_DIR=${root}/adm
+    RUN_DIR=${root}/etc
     EX_UXMON_DIR=${ex_prefix}/uxmon
     EX_BIN_DIR=${ex_prefix}/bin
     EX_VAR_DIR=${ex_prefix}/var
     EX_ETC_DIR=${ex_prefix}/etc
-    EX_ADM_DIR=${ex_prefix}/adm
+    EX_ADM_DIR=${ex_prefix}/etc
     EX_WEB_DIR=${ex_prefix}/www
     EX_CGI_DIR=${ex_prefix}/cgi
     EX_DOC_DIR=${ex_prefix}/doc
     EX_USER_SBIN_DIR=${EX_BIN_DIR}
     EX_USER_BIN_DIR=${EX_BIN_DIR}
-    EX_RUN_DIR=${ex_prefix}/adm
+    EX_RUN_DIR=${ex_prefix}/etc
     EX_MAN_DIR=${ex_prefix}/man
 fi
 
